sepa_parser.gemspec

```diff
@@ -28,13 +29,7 @@ Currently pain.001.001.03 is supported.'
   spec.bindir = 'exe'
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 3.0'
 
   spec.add_dependency 'rexml'
-
-  spec.add_development_dependency 'bundler', '~> 2.0'
```

Comment:
Waarom is dit verplaatst? Volgens mij maakt het niet echt uit of dit hier staat of in de Gemfile dus beide zijn goed wat mij betreft.

Reply:
Rubocop begon er over te klagen. Die vond dat development dependencies niet onderdeel zijn van de gem dus hier niet hoeven te staan. Ik houd Rubocop graag te vriend.